### PR TITLE
Fix Edit profile tests

### PR DESCRIPTION
**Pull request by bebef1987**
this should fix the edit profile tests

Please review and give feedback

### CODE REVIEW

**Comment by ioana-chiorean** [comment]
Maybe some Mozillians can help here? @m8ttyB ? @davehunt ?

**Comment by m8ttyB** [comment]
Looks like travis failed, there are a few flake8 bits that need to be cleaned up.

File: pages/settings.py [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+
+class Settings(Base):
+    _profile_tab_locator = (By.ID, "profile-tab")
+    _profile_button_locator = (By.CSS_SELECTOR, "#profile-li > a")
+
+    _you_and_mozilla_tab_locator = (By.ID, "youandmozilla-tab")
+    _you_and_mozilla_button_locator = (By.CSS_SELECTOR, "#youandmozilla-li > a")
+
+    _developer_tab_locator = (By.ID, "developer-tab")
+    _developer_button_locator = (By.CSS_SELECTOR, "#developer-li > a")
```
Comment (m8ttyB) [inline pages/settings.py]: Nit, for consistency across the other page objects, use single quotes, `'`.

File: pages/settings.py [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+        
+            def select_random_month(self):
+                return self.select_month(random.choice(self.months_values[1:]))
+
+            def select_random_year(self):
+                return self.select_year(random.choice(self.years_values[1:]))
+
+            @property
+            def years_values(self):
+                return [year.get_attribute('value') for year in self._root_element.find_elements(*self._year_locator)]
```
Comment (m8ttyB) [inline pages/settings.py]: It's not clear the difference between `def year` and `def years_values`. At minimum, add code comments clarifying their usage.

**Comment by AndreiH** [comment]
Tested this on WIndows 10 FF 42, all tests pass. @bebef1987 can you update the PR according to @m8ttyB's review? thanks!

File: pages/settings.py (hunk 1 above)
Comment (bebef1987) [inline pages/settings.py]: Done

File: pages/settings.py (hunk 2 above)
Comment (bebef1987) [inline pages/settings.py]: Updated added code comments

**Comment by comzeradd** [comment]
Shouldn't the tests regarding registration be updated? For instance the [test_creating_profile_without_checking_privacy_policy_checkbox](https://github.com/bebef1987/mozillians-tests/blob/fix_edit_profile/tests/test_profile.py#L105) fails locally because the profile creation form doesn't contain skills any more.

**Comment by bebef1987** [comment]
@comzeradd That should be in an new commit as it's an new page and modifies other files.
I want to fix the edit profile tests and then continue with the new profile ones

File: tests/test_profile.py [hunk 3]
```diff
@@ -256,7 +277,7 @@ def test_that_user_can_create_and_delete_group(self, base_url, selenium, vouched
 
         home_page = Home(base_url, selenium)
         home_page.login(vouched_user['email'], vouched_user['password'])
-        edit_page = home_page.header.click_edit_profile_menu_item()
+        edit_page = home_page.header.click_settings_menu_item()
         groups = edit_page.click_find_group_link()
```
Comment (comzeradd) [inline tests/test_profile.py]: This fails locally since the new Settings class has no longer a `click_find_group_link` method.

**Comment by comzeradd** [comment]
I fixed the registration and groups tests. So we can close this in favor of #205.

**Comment by AndreiH** [comment]
This is ok with me, I'll give it an r+. tests are passing on dev now. @m8ttyB what do you think?

**Comment by comzeradd** [comment]
@AndreiH if you have the time for review I think it makes more sense to merge #205.

**Comment by johngian** [comment]
Hey @bebef1987 can we go ahead and close this PR after #205 ?

**Comment by johngian** [comment]
I suggest we close this one after #205 
@bebef1987 what do you think?

**Comment by bebef1987** [comment]
Sounds good to me

**Pull request closed without merging**